binr/rax2/rax2.c

```diff
@@ -353,7 +353,8 @@ static int rax(char *str, int len, int last) {
 		return true;
 	} else if (flags & (1 << 12)) { // -E
 		const int len = strlen (str);
-		char *out = calloc (sizeof (char), ((len + 1) * 4) / 3);
+		/* http://stackoverflow.com/questions/4715415/base64-what-is-the-worst-possible-increase-in-space-usage */
+		char *out = calloc (sizeof (char), (len + 2) / 3 * 4 + 1); // ceil(len/3)*4 plus 1 for NUL
```

Comment:
i have the feeling that this allocation logic should be done inside the r_base64* apis